bolt/spark/array.py

```diff
@@ -568,6 +570,42 @@ def swapaxes(self, ax1, ax2):
 
         return self.transpose(p)
 
+    def reshape(self, *new):
```

Comment:
I took this from your `transpose` function on the `Shapes` objects. Should we change it there too then?